### PR TITLE
Run conformance test in the presubmit test

### PR DESCRIPTION
**Pull request by ZhiminXiang**

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -58,6 +71,11 @@ required = [
   unused-packages = true
   non-go = true
 
+[[prune.project]]
+  name = "knative.dev/serving"
+  non-go = false
+  go-tests = false
+
```
Comment (mattmoor) [inline Gopkg.toml:78]: Remove trailing whitespace:
```suggestion

```

File: test/e2e-common.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  echo ">> Creating test resources (test/config/)"
+  ko apply ${KO_FLAGS} -f test/config/ || return 1
+
+  # Bringing up controllers.
+  echo ">> Bringing up Cert-Manager"
+  local dir=third_party/cert-manager-0.12.0
+  kubectl apply -f ${dir}/cert-manager-crds.yaml || return 1
+  kubectl apply -f ${dir}/cert-manager.yaml || return 1
+  wait_until_batch_job_complete cert-manager || return 1
+
```
Comment (mattmoor) [inline test/e2e-common.sh:46]: Remove trailing whitespace:
```suggestion

```

File: test/e2e-common.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+function add_trap() {
+  local cmd=$1
+  shift
+  for trap_signal in $@; do
+    local current_trap="$(trap -p $trap_signal | cut -d\' -f2)"
+    local new_cmd="($cmd)"
+    [[ -n "${current_trap}" ]] && new_cmd="${current_trap};${new_cmd}"
+    trap -- "${new_cmd}" $trap_signal
+  done
+}
```
Comment (mattmoor) [inline test/e2e-common.sh:65]: Add trailing newline:
```suggestion
}

```

File: test/e2e-test.sh [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+source $(dirname $0)/e2e-common.sh
+
+
+# Script entry point.
+initialize $@  --skip-istio-addon
+
+go_test_e2e -timeout=20m -parallel=12 ./vendor/knative.dev/serving/test/conformance/certificate || fail_test
+
+success
```
Comment (mattmoor) [inline test/e2e-test.sh:25]: Add trailing newline:
```suggestion
success

```

File: test/upload-test-images.sh [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        `# ko resolve is being used for the side-effect of publishing images,` \
+        `# so the resulting yaml produced is ignored.` \
+        | ko resolve --strict ${tag_option} -RBf- > /dev/null
+    done
+  )
+}
+
+: ${KO_DOCKER_REPO:?"You must set 'KO_DOCKER_REPO', see DEVELOPMENT.md"}
+
+upload_test_images $@
```
Comment (mattmoor) [inline test/upload-test-images.sh:43]: Add trailing newline:
```suggestion
upload_test_images $@

```

File: third_party/cert-manager-0.12.0/download-cert-manager.sh [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+# Copy the CRD yaml file
+cp deploy/manifests/00-crds.yaml ../cert-manager-crds.yaml
+)
+
+# Download the cert-manager yaml file
+wget $YAML_URL
+
+# Clean up.
+rm -rf cert-manager-${CERT_MANAGER_VERSION}
+rm v${CERT_MANAGER_VERSION}.tar.gz
```
Comment (mattmoor) [inline third_party/cert-manager-0.12.0/download-cert-manager.sh:40]: Add trailing newline:
```suggestion
rm v${CERT_MANAGER_VERSION}.tar.gz

```

File: Gopkg.toml (hunk 1 above)
Comment (mattmoor) [inline Gopkg.toml:78]: Remove trailing whitespace:
```suggestion

```

**Comment by tcnghia** [comment]
> The following users are mentioned in OWNERS file(s) but are not members of the knative org.
> 
> Once all users have been added as members of the org, you can trigger verification by writing `/verify-owners` in a comment.
> 
> * networking-reviewers
>   
>   * test/OWNERS
> * networking-approvers
>   
>   * test/OWNERS

Please fix ^

**Comment by ZhiminXiang** [comment]
> > The following users are mentioned in OWNERS file(s) but are not members of the knative org.
> > Once all users have been added as members of the org, you can trigger verification by writing `/verify-owners` in a comment.
> > 
> > * networking-reviewers
> >   
> >   * test/OWNERS
> > * networking-approvers
> >   
> >   * test/OWNERS
> 
> Please fix ^

Done.

**Comment by ZhiminXiang** [comment]
/hold

I guess we probably need the PR https://github.com/knative/test-infra/pull/1903 to be merged

File: cmd/controller/main.go [hunk 7]
```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2019 The Knative Authors
```
Comment (mattmoor) [inline cmd/controller/main.go:2]: Incorrect copyright year:
```suggestion
Copyright 2020 The Knative Authors
```

**Comment by ZhiminXiang** [comment]
I was able to make the conformance test pass. But I need to copy the cmd files from serving repo.
@tcnghia  could you help to fix the CLA? :)

**Pull request merged**